### ISSUE TITLE
A11y_AzureDataStudioSQLMigration_AzureSQLMigration_Orientation:The "Help articles and link" section is not clearly visible in Potrait mode.

### ISSUE THREAD

**Issue by Shivamd1** [microsoft/azuredatastudio#15737]
[“Check out Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

**GitHubTags:** #A11yWCAG2.1;#A11yMAS;#A11yTCS;#A11ySev2;#SH_AZUREDATASTUDIOSQLMigration_June2021;#SQL Azure Data Studio;#Win10;#FTP;#DesktopApp;#WCAG1.3.4;#AINotInScope;#Orientation

### Pre-Requisite:
Change the Display orientation to Potrait.

### Environment Details:
Application Name: SQL Azure Data Studio
Windows Version: Win10

### Repro Steps:

1. Launch the application-Azure Data Studio-Insiders and install extension Azure SQL Migration.
2. Connect to any sql server.
3. Tab till Azure SQL migration Tab and press enter.
4. View the page in Potrait mode.

### Actual:
The "Help articles and link" section is not clearly visible in Potrait mode.

### Expected:
All the elements in the page should be clearly visible in Potrait mode.

### User Impact:
Users viewing the page in Potrait mode will not get the complete information.

**Comment by aasimkhan30** [comment]
Similar to #15736

**Comment by Yashi1201** [comment]
Closing the bug as the issue is not repro 
Version : 1.42.0-insider(user-setup)
![15737_issue fixed](https://user-images.githubusercontent.com/93374043/218383215-a5522f06-7e0d-4dd8-b34f-5bfbc93b9bd2.png)

**GithubTags**:#Closed;
